spec.html

```diff
@@ -39927,6 +39951,8 @@ <h1>NewPromiseReactionJob ( _reaction_, _argument_ )</h1>
           1. If _reaction_.[[Handler]] is not *undefined*, then
             1. Let _getHandlerRealmResult_ be GetFunctionRealm(_reaction_.[[Handler]]).
             1. If _getHandlerRealmResult_ is a normal completion, then set _handlerRealm_ to _getHandlerRealmResult_.[[Value]].
+            1. Else, set _handlerRealm_ to the current Realm Record.
+            1. NOTE: _handlerRealm_ is never *null* when the handler is not *undefined*. When the handler is a revoked Proxy and no code runs, _handlerRealm_ is used to create error objects.
```

Comment:
```suggestion
            1. NOTE: _handlerRealm_ is never *null* unless the handler is *undefined*. When the handler is a revoked Proxy and no code runs, _handlerRealm_ is used to create error objects.
```

Reply:
Also, "no code runs" is somewhat unclear. Should this be "no ECMAScript code"?

Reply:
Will change to "no ECMAScript code"